TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -488,7 +486,6 @@ private int getScreenVersion() {
     }
 
     private boolean portIsFree(int port) {
```

Comment:
Please rename the method to `isPortFree` in order to stick to coding conventions for boolean methods